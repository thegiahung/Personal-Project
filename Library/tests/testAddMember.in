Add member Python
Add member Java
Add member Ruby
Add member Atlas
Member 100000
Member 100001
Member 100002
Member 100003
Member 100004
Exit
