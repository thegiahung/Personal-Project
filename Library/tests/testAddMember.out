user: Success.

user: Success.

user: Success.

user: Success.

user: 100000: Python

user: 100001: Java

user: 100002: Ruby

user: 100003: Atlas

user: No such member in system.

user: Ending Library process.
